.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "framework"]
 	path = framework
 	url = git@github.com:venediktov/vanilla-rtb.git
```

Comment:
I propose to change it to https://  - travis is failing 
url = "https://github.com/venediktov/vanilla-rtb.git"